mkdocs/docs/HPC/alphafold.md

```diff
@@ -0,0 +1,232 @@
+
+AlphaFold DB provides open access to over 200 million protein.
+
+See <https://www.vscentrum.be/alphafold> for more information and there you can also find a getting started video recording if you prefer that. 
+
+## Documentation & extra material
+
+This chapter focuses specifically on the use of AlphaFold on the {{hpcinfra}}.
+It is intented to augment the existing AlphaFold documentation rather than replace it.
+It is therefore recommended to first familiarize yourself with AlphaFold. The following resources can be helpful:
+
+- AlphaFold website: <https://alphafold.com/>
+- AlphaFold repository: <https://github.com/deepmind/alphafold/tree/main>
+- AlphaFold FAQ: <https://alphafold.com/faq>
+- VSC webpage about AlphaFold: <https://www.vscentrum.be/alphafold>
+- Recording of "Getting started with AlphaFold" by Kenneth Hoste, HPC-Ugent (also available on the [VSC webpage](https://www.vscentrum.be/alphafold)).
+    - [slides](https://www.vscentrum.be/_files/ugd/5446c2_f19a8723f7f7460ebe990c28a53e56a2.pdf?index=true)
+
+
+## Using AlphaFold on {{hpcinfra}}
+
+Several different versions of AlphaFold are installed on both the CPU and GPU HPC-UGent Tier-2 clusters, see the output of `module avail AlphaFold`.
+If running this command on a GPU-cluster, extra CUDA modules will show up:
+
+```shell
+$ module avail AlphaFold
+
+------------ /apps/gent/RHEL8/cascadelake-volta-ib/modules/all -------------
+   AlphaFold/2.0.0-fosscuda-2020b
+   AlphaFold/2.1.1-fosscuda-2020b
+   AlphaFold/2.1.2-foss-2021a-CUDA-11.3.1
+   AlphaFold/2.2.2-foss-2021a-CUDA-11.3.1
+   AlphaFold/2.3.0-foss-2021b-CUDA-11.4.1
+   AlphaFold/2.3.1-foss-2022a-CUDA-11.7.0
+
+--------------- /apps/gent/RHEL8/cascadelake-ib/modules/all ----------------
+   AlphaFold/2.0.0-foss-2020b    AlphaFold/2.3.1-foss-2022a
+   AlphaFold/2.1.2-foss-2021a    AlphaFold/2.3.4-foss-2022a-ColabFold (D)
+   AlphaFold/2.2.2-foss-2021a
+
+```
+
+load the module:
+
+```shell
+ml load AlphaFold
+```
+
+!!! Tip "You should specify a version for better reproducibility."
+
+!!! Warning
+    
+    When using AlphaFold, you should submit jobs to a GPU cluster for better performance, see [GPU clusters](gpu.md).
+    Later in this chapter, you will find a comparison between running AlphaFold on CPUs or GPUs.
+
+The large database (~4.5TB) that is also required to run AlphaFold has been made available on the HPC-UGent infrastructure in a central location.
+So that you don't have to download that data yourself.
+
+Under: `{{directory}}`
+
+```shell
+$ ls /arcanine/scratch/gent/apps/AlphaFold
+20210812  20211201  20220701  20230310
+```
+
+The directories located there indicate when the data was downloaded, so that this leaves room for providing updated datasets later.
+
+As of writing this documentation the latest version is `20230310`.
+
+One important remark here is that the filesystem where this AlphaFold dataset is located is *not* available on the default victini cluster,
```

Comment:
This is correct, but let's exclude this, since `victini` will only be around for a couple more months (and new cluster will have access to `/arcanine/scratch`, so there won't be exceptions anymore)